### DISCUSSION TITLE
Wrong Browser Url in Livewire 2.6.0

### DISCUSSION THREAD

**Discussion by michele-grifa** [livewire/livewire#3735] [7. Bugs (archived)] [answered]
I have a component with the following query string that works well with 2.5.0 release
```php
protected $queryString = [
    'page' => ['except' => 1],
    'sort' => ['except' => ''],
    'queryArticolo' => ['except' => ''],
    'queryCategoria1' => ['except' => ''],
    'queryCategoria2' => ['except' => ''],
    'brand' => ['except' => ''],
    'queryFornitore' => ['except' => '']
];
```

After the 2.6.0 release when the component load, the browser url should look like this
`http://localhost:8080/logistica/inventario?sort=giacenza_complessiva_desc` 

instead the url looks like this
`http://localhost:8080/?sort=giacenza_complessiva_desc`

**Comment by joshhanley** [comment]
@michele-grifa this issue has already been reported on #3725, so we will keep the conversation there 🙂

Thanks!

**Reply by michele-grifa** [reply]
Sorry, i reported without checking.

**Reply by joshhanley** [reply]
@michele-grifa no worries 🙂 I'm struggling to replicate the issue, so if you can provide any insight on that thread or #3734 that would be much appreciated, so I can try and get it fixed!

**Reply by georgeboot** [reply]
@joshhanley If you want me to demo the issue with a screen share or so, feel free to ping.

**Reply by joshhanley** [reply]
@georgeboot thanks! I've managed to track it down 🙂

**Comment by joshhanley** [comment, marked as answer]
I have tagged v2.6.2 which fixes this issue 🙂

Please test it and let us know if it's still happening!

Thanks!
